src/components/FamilyOrInsureePoliciesSummary.js

```diff
@@ -249,7 +249,7 @@ class FamilyOrInsureePoliciesSummary extends PagedDataHandler {
     let a = [
       this.sorter("productCode"),
       this.sorter("productName"),
-      this.sorter("enrolmentDate"),
+      this.sorter("effectiveDate"),
```

Comment:
Do not replace it, it was requsted by Nepal in this ticket:
https://openimis.atlassian.net/browse/ONI-18
I may have made a mistake there, but if anything both enroll and effective date should be visible.

Reply:
Thanks! I restored enroll date.